backend/app/resources/create_user_info_dto.py

```diff
@@ -0,0 +1,28 @@
+import re
+
+emailRegex = r"^[a-z0-9]+[\._]?[a-z0-9]+[@]\w+[.]\w{2,3}$"
+phoneRegex = r"^[0-9]{9}$"
+
+
+class CreateUserInfoDTO:
+    def __init__(self, **kwargs):
+        self.contact_name = kwargs.get("contact_name")
+        self.contact_email = kwargs.get("contact_email")
+        self.contact_phone = kwargs.get("contact_phone")
+        self.role = kwargs.get("role")
+
+    def validate(self):
+        error_list = []
+        if type(self.contact_name) is not str:
+            error_list.append("The contact name supplied is not a string.")
+        if type(self.contact_email) is not str or not re.search(
+            emailRegex, self.contact_email
+        ):
+            error_list.append("The contact email supplied is not a string.")
```

Comment:
Might be good to change this error message if the error is due to not matching the regex (format mismatch) instead of a type mismatch.